experimental/Schemes/SimplifiedAffineScheme.jl

```diff
@@ -21,6 +21,9 @@ a `SimplifiedAffineScheme` ``Y`` with ``X`` as its `original`.
 from the one of `X` and hence the two schemes will not compare using `==`.
 """
 function simplify(X::AbsAffineScheme{<:Field})
+  # We need to avoid creating a polynomial ring with zero variables
+  dim(X) == 0 && vector_space_dimension(OO(X)) == 0 && return SimplifiedAffineScheme(X, X, identity_map(X), identity_map(X), check=false)
```

Comment:
May I suggest moving the point to `0`? There is the function `rational_point_coordinates`. 
It will push some of the complexity into the gluings ... but things like saturation should be easier. 
Not sure though how exactly `simplify` is hooked into everything.

Reply:
That's actually not that easy to program. I don't think it pays off since these schemes are already very simple and stuff like saturation will not be applied here.